src/components/NftRewards/NftTierCard.tsx

```diff
@@ -71,6 +74,24 @@ export function NftTierCard({
     }
   }
 
+  const contentType = useContentType(fileUrl)
+  const isVideo = contentType === 'video/mp4'
```

Comment:
**suggestion** because `video/mp4` is used twice, lets make it a constant somewhere.